sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -253,9 +251,10 @@ private[sbtplugin] object ScalaJSPluginInternal {
 
             val out = linkerImpl.outputDirectory(outputDir.toPath)
 
+            val tlog = scalaJSLoggerFactory.value(log)
```

Comment:
This should go outside the `FileFunction.cached` to please sbt (at least the `scalaJSLoggerFactory.value` part of it): `.value` calls are supposed to go outside of control-flow and closures.